solution/ui/regulations/eregs-vite/src/components/resources/ResourceExportBtn.vue

```diff
@@ -1,12 +1,21 @@
 <template>
-    <div class="resource_btn_container">
```

Comment:
I figured I would leave some comments on your commit to give you an idea of what I changed and why.

Reply:
Sounds good